app/models/grda_warehouse/census/project_batch.rb

```diff
@@ -49,16 +38,36 @@ def build_census_batch
       end
     end
 
-    def add_clients_to_census_buckets (collection, column_name)
-      collection.each do | (date, project_id), count |
+    def inventory_active_on_date?(inventory, date)
+      # Always active if all dates are blank
+      if inventory.InformationDate.blank? &&
+         inventory.InventoryStartDate.blank? &&
+         inventory.InventoryEndDate.blank?
+        return true
+      end
+
+      # Determine the effective start date (InfoDate takes precedence)
+      start_date = inventory.InformationDate || inventory.InventoryStartDate
```

Comment:
This doesn't seem correct.  `InventoryStartDate` is the date the inventory started being active, `InformationDate` is the date when the current state of the record was known.

I think maybe we were trying to infer a start date when none was present previously.  If we don't know when it started, we at least know when it was last accurate, but I think we should just ignore `InformationDate` when determining the start date.  It can probably be pulled out of line 43 above as well.

@gigxz does that make sense to you?

Reply:
Yeah I agree

Reply:
Correct, I believe the existing logic is flawed.

Reply:
adjusted, the logic no longer involves InformationDate